README.md

```diff
@@ -34,3 +34,24 @@ The hub is a long term data store.
 Run ./gradlew build
 
 The above command compiles the code, runs the unit tests and integration tests.~~~~
+
+
+### How to run System tests:
+
+From Local:
+
+1) Setup hub:
+    docker run -p 80:80 flightstats/hub:latest
```

Comment:
minor nitpick, but I think this should be something like ... flightstats/hub:local. The image created by the docker shell script is name spaced local because we used to call everything 'latest' which caused some confusing conflicts if you pulled the image from our docker repo, which is more rightly called latest